app/services/analytics_events.rb

```diff
@@ -2952,6 +2952,37 @@ def idv_selfie_image_added(
   end
   # rubocop:enable Naming/VariableName,Naming/MethodParameterName
 
+  # rubocop:disable Naming/VariableName,Naming/MethodParameterName,
+  # @param [Boolean] acuant_sdk_upgrade_a_b_testing_enabled
+  # @param [String] acuant_version
+  # @param [String] flow_path whether the user is in the hybrid or standard flow
+  # @param [Boolean] isDrop
+  # @param [String] source
+  # @param [String] use_alternate_sdk
+  # @param [Boolean] liveness_checking_required
+  def idv_selfie_image_clicked(
+    acuant_sdk_upgrade_a_b_testing_enabled:,
+    acuant_version:,
+    flow_path:,
+    isDrop:,
+    source:,
+    use_alternate_sdk:,
+    liveness_checking_required: nil,
+    **_extra
```

Comment:
Is `captureAttempts` easy to add in?

Reply:
> Is `captureAttempts` easy to add in?

👍 I'd hope we could add all of the properties here we're expecting to be logged.